src/slam_karto.cpp

```diff
@@ -1297,6 +1347,17 @@ SlamKarto::updateMap()
     // Publish the map
     sst_.publish(map_.map);
     sstm_.publish(map_.map.info);
+
+    // Publish the new graph
+    std::swap(graph_msg_, graph_msg);
```

Comment:
Any specific reason why you are using swap instead of move here? Performance reasons?

Reply:
No...just had "swap" in my head at the time. Will fix.